### PR TITLE
feat: return for `wrap`/`unwrap`

### PR DESCRIPTION
**Pull request by deluca-mike**

### CODE REVIEW

File: src/WrappedMToken.sol [hunk 1]
```diff
@@ -545,9 +545,10 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
      * @dev   Wraps `amount` M from `account_` into wM for `recipient`.
      * @param account_   The account from which M is deposited.
      * @param recipient_ The account receiving the minted wM.
-     * @param amount_    The amount of M deposited and wM minted.
+     * @param amount_    The amount of M deposited.
+     * @param wrapped_   The amount of wM minted.
```
Comment (PierrickGT) [inline src/WrappedMToken.sol:549]:
```suggestion
     * @return wrapped_   The amount of wM minted.
```

File: src/WrappedMToken.sol [hunk 2]
```diff
@@ -557,20 +558,25 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
...
+        _mint(recipient_, wrapped_ = UIntMath.safe240(IMTokenLike(mToken).balanceOf(address(this)) - startingBalance_));
     }
 
     /**
      * @dev   Unwraps `amount` wM from `account_` into M for `recipient`.
      * @param account_   The account from which WM is burned.
      * @param recipient_ The account receiving the withdrawn M.
-     * @param amount_    The amount of wM burned and M withdrawn.
+     * @param amount_    The amount of wM burned.
+     * @param unwrapped_ The amount of M withdrawn.
```
Comment (PierrickGT) [inline src/WrappedMToken.sol:569]:
```suggestion
     * @return unwrapped_ The amount of M withdrawn.
```

File: src/interfaces/IWrappedMToken.sol [hunk 3]
```diff
@@ -90,28 +90,32 @@ interface IWrappedMToken is IMigratable, IERC20Extended {
     /**
      * @notice Wraps `amount` M from the caller into wM for `recipient`.
      * @param  recipient The account receiving the minted wM.
-     * @param  amount    The amount of M deposited and wM minted.
+     * @param  amount    The amount of M deposited.
+     * @param  wrapped   The amount of wM minted.
```
Comment (PierrickGT) [inline src/interfaces/IWrappedMToken.sol:94]:
```suggestion
     * @return  wrapped   The amount of wM minted.
```

File: src/interfaces/IWrappedMToken.sol [hunk 4]
```diff
@@ -90,28 +90,32 @@ interface IWrappedMToken is IMigratable, IERC20Extended {
...
      */
-    function wrap(address recipient, uint256 amount) external;
+    function wrap(address recipient, uint256 amount) external returns (uint240 wrapped);
 
     /**
      * @notice Wraps all the M from the caller into wM for `recipient`.
      * @param  recipient The account receiving the minted wM.
+     * @param  wrapped   The amount of wM minted.
```
Comment (PierrickGT) [inline src/interfaces/IWrappedMToken.sol:101]:
```suggestion
     * @return  wrapped   The amount of wM minted.
```

File: src/interfaces/IWrappedMToken.sol [hunk 5]
```diff
@@ -90,28 +90,32 @@ interface IWrappedMToken is IMigratable, IERC20Extended {
...
      */
-    function wrap(address recipient) external;
+    function wrap(address recipient) external returns (uint240 wrapped);
 
     /**
      * @notice Unwraps `amount` wM from the caller into M for `recipient`.
      * @param  recipient The account receiving the withdrawn M.
-     * @param  amount    The amount of wM burned and M withdrawn.
+     * @param  amount    The amount of wM burned.
+     * @param  unwrapped The amount of M withdrawn.
```
Comment (PierrickGT) [inline src/interfaces/IWrappedMToken.sol:109]:
```suggestion
     * @return  unwrapped The amount of M withdrawn.
```

File: src/interfaces/IWrappedMToken.sol [hunk 6]
```diff
@@ -90,28 +90,32 @@ interface IWrappedMToken is IMigratable, IERC20Extended {
...
      */
-    function unwrap(address recipient, uint256 amount) external;
+    function unwrap(address recipient, uint256 amount) external returns (uint240 unwrapped);
 
     /**
      * @notice Unwraps all the wM from the caller into M for `recipient`.
      * @param  recipient The account receiving the withdrawn M.
+     * @param  unwrapped The amount of M withdrawn.
```
Comment (PierrickGT) [inline src/interfaces/IWrappedMToken.sol:116]:
```suggestion
     * @return  unwrapped The amount of M withdrawn.
```

**Review by PierrickGT — CHANGES_REQUESTED** [review]

**Review by toninorair — APPROVED** [review]

File: src/WrappedMToken.sol (hunk 1 above)
Comment (deluca-mike) [reply src/WrappedMToken.sol:549]: Good catches. Seems like the compiler does not catch this.

**Review by PierrickGT — APPROVED** [review]

**Pull request merged**